### ISSUE TITLE
[Feature Request]: Make disabled Hop lines more visible. For example: 50% transparent and "dotted/divided".

### ISSUE THREAD

**Issue by Mattang-Dan** [apache/hop#3581]
### What would you like to happen?

1) Disabled Hop lines are 50% transparent and "dotted/divided" so they are more visible. (It is very hard to see disabled Hop lines.)

2) Have an option to have the Hop lines twice as wide/thick. (less important)

### Issue Priority

Priority: 3

### Issue Component

Component: Hop Gui

**Comment by nadment** [comment]
Is it more visible like this?

![image](https://github.com/apache/hop/assets/39568358/7b4ecc8e-83b1-41c5-b969-8f6a88a1f89c)

**Comment by hansva** [comment]
doesn't this add confusion between de error hop though

**Comment by Mattang-Dan** [comment]
I really like how the disabled dotted lines look. If a disabled line, I think it would look great if more 'faded' (maybe 50% transparent?), as that red disabled line looks very much at the forefront.

**Comment by hansva** [comment]
I would be careful with too much transparency, as the hop might no longer be visible to people with visual impairment.

**Comment by Mattang-Dan** [comment]
True. remove the word 'transparency', and just a more faded color for disabled lines. I will try to send an example.

**Comment by Mattang-Dan** [comment]
![Disabled_Hop_Lines](https://github.com/apache/hop/assets/100213981/31bf4723-aa47-465a-bbab-c4ea1c6e2a4f)

**Comment by nadment** [comment]
For me, visibility is less good

**Comment by hansva** [comment]
In my personal opinion the icons on the hops speak for themselves when the hops are disabled. We could bring this subject to the users mailing list and have more input on the subject before taking a decision.
